bestpy/answers.py

```diff
@@ -1,34 +1,3 @@
-import platform
-
-from typing import Type
-
-
-def add_from_class(cls: Type, target_dict: dict) -> None:
-    """Adds the methods of the specified class to the target dict, if it doesn't start with '_'."""
-    method_names = [
-        name for name in dir(cls) if not name.startswith("_")
-    ]  # Only methods that don't start with _
-    answer_methods = {name: getattr(cls, name) for name in method_names}
-    target_dict.update(answer_methods)
-
-
-class CodeAnswers:
-    """Class for answers that require code execution on every access"""
-
-    def __init__(self):
-        pass
-
-    @staticmethod
-    def os() -> str:
-        name = platform.system()
-
-        # Replacing "Darwin", since it might be confusing for macOS users
-        if name == "Darwin":
-            name = "macOS"
-
-        return name
```

Comment:
Did this completely vanish? Any reason for that? I think the variable inputs, that change depending on the user, are quite fun to have.